dscntou/solution-challenge-frontend
# Add several pages(login, sign_up, verify, home).

## Types of changes


## Description

I add some pages in flutter app and all of their exterior will change possibly.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Make sure that you're already installed flutter and connected mobile device.
```
solution-challenge-frontend$ flutter doctor -v
```
2. Go to flutter project directory.
```
solution-challenge-frontend$ cd src/app
solution-challenge-frontend/src/app$ 
```
3. Install dependencies.
```
solution-challenge-frontend/src/app$ flutter pub get
```
4. Install app to your mobile device.
```
solution-challenge-frontend/src/app$ flutter install
```
5. Open the app which installs on your mobile device.

## Expected behavior

+  Login Page: An email and password input field, a sign up button to go to sign up page and a login button to go to home page.
+ Sign up page: Four fields need to fill, and two password fields must be same.
+ Verify page: Four numeric input fields and don't have any function.
+ Home page: Just for preview when pressed login button.

## More Information

+ All buttons are just for preview, they don't have function actually.
+ Home page isn't completed.
